src/k8s-engine/lima.ts

```diff
@@ -306,6 +295,11 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
    */
   protected currentAction: Action = Action.NONE;
 
+  protected writeSetting(changed: RecursivePartial<typeof this.cfg>) {
+    mainEvents.emit('settings-write', { kubernetes: changed });
```

Comment:
A warning that this change might not live a long time -- Jan has expressed a desire to move non-kubernetes properties out of the `kubernetes` block (like container-engine). I haven't looked closely, but at some point we might want to deal with the config subset from root rather than from `kubernetes`.

*NOT AN ACTIONABLE COMMENT*

Reply:
Yep, that's expected — and we will need to deal with that in the current code either way, so it shouldn't be an issue (this will just change to take a `RecursivePartial<Settings>` instead).